lib/exercism/user.rb

```diff
@@ -80,6 +90,10 @@ def self.find_by_username(username)
     find_by(username: username)
   end
 
+  def self.find_by_username_and_share_key(username, key)
+    find_by(username: username, share_key: key)
+  end
```

Comment:
@tejasbubane :+1: I went ahead and removed the inline method as it was only being called once.